ui/src/timeMachine/actions/queries.ts

```diff
@@ -106,7 +106,14 @@ const isFromBucket = (node: Node) => {
   )
 }
 
-export const executeQueries = () => async (dispatch, getState: GetState) => {
+export const setQueryToLoading = () => dispatch => {
```

Comment:
I don't understand why this is a thunk and not just an action?

Reply:
Totally can be. Good call